.changeset/pre.json

```diff
@@ -0,0 +1,10 @@
+{
+  "mode": "pre",
```

Comment:
this configures our release automation to only do beta releases for now

Reply:
How exactly does that work here? Will changeset pick a version for us? And if so, will it be a beta release like `1.0.0-beta.0`, or will it be 0.x prereleases like `0.1.0`?

Reply:
not sure when things are pre 1.0 -- I assume it'll do 1.0.0-beta.x until we turn off this mode. It'll also update the `beta` tag on npm -- I did a test with this on ember-toucan-styles, but that was for a 2.0.0 beta that I ended up abandoning